build.xml

```diff
@@ -37,8 +38,8 @@
   <property name="pgsqlhost" value="localhost" />
   <property name="pgsqlrootuser" value="postgres" />
   <property name="disable_csp" value="false" /> <!-- set value to "true" to disable Content Security Policy -->
-  <property name="phpunit_command" value="${srcdir}/vendor/bin/phpunit" /><!-- if you have issues with segfaults during code coverage generation, try adding -dzend.enable_gc=0 to this (but this fix was probably only needed in PHP 5) -->
-  <property name="phpunit_extra_params" value="" />
+  <property name="phpunit_command" value="${srcdir}/vendor/bin/phpunit" />
+  <property name="phpunit_extra_params" value="" /><!-- if you have issues with segfaults during code coverage generation, try adding -dzend.enable_gc=0 to this (but this fix was probably only needed in PHP 5) -->
```

Comment:
Should we drop the zend.enable_gc comment entirely? I suspect it no longer applies...

Reply:
Yes, dropped!